.devcontainer/devcontainer.json

```diff
@@ -8,17 +8,23 @@
   // Features to add to the dev container. More info: https://containers.dev/features.
   "features": {
     "ghcr.io/devcontainers-contrib/features/pnpm": {
-      "version": "9.1.3"
+      "version": "9.8.0"
     },
-    "ghcr.io/joshuanianji/devcontainer-features/mount-pnpm-store": {}
+    "ghcr.io/joshuanianji/devcontainer-features/mount-pnpm-store": {},
+    "ghcr.io/devcontainers-contrib/features/npm-package": {
+      "package": "radashi-helper",
+      "version": "latest"
+    }
```

Comment:
Let's add `radashi-helper` to the package.json instead, so it's available even to people not using dev containers.

Probably best not to use `latest` as it might increase risk of supply chain attacks if one of our transient dependencies ever gets compromised. Once we have Renovate up and running, periodic updates should be good enough.

Reply:
We can add it, but even when Radashi is included only as a development dependency, we still can't use the `radashi` keyword before adding `pnpm` prefix or similar. By configuring this in the Dev Container, anyone using it won't need to run `pnpm radashi *`. 

Of course, there are some workarounds we could implement only with the package.json, but in this case, it would be better to just use `pnpm` prefix.

About the version, I agree with you.

![image](https://github.com/user-attachments/assets/af9360a6-76ff-4911-8aab-14a633a2e923)